NorthstarDedicatedTest/version.cpp

```diff
@@ -0,0 +1,59 @@
+#include "version.h"
+#include "pch.h"
+
+char version[16];
+char NSUserAgent[32];
+
+void InitialiseVersion() {
+	HRSRC hResInfo;
+	DWORD dwSize;
+	HGLOBAL hResData;
+	LPVOID pRes, pResCopy;
+	UINT uLen = 0;
+	VS_FIXEDFILEINFO* lpFfi = NULL;
+	HINSTANCE hInst = ::GetModuleHandle(NULL);
+
+	hResInfo = FindResource(hInst, MAKEINTRESOURCE(1), RT_VERSION);
+	if (hResInfo != NULL) {
+		dwSize = SizeofResource(hInst, hResInfo);
+		hResData = LoadResource(hInst, hResInfo);
+		if (hResData != NULL) {
+			pRes = LockResource(hResData);
+			pResCopy = LocalAlloc(LMEM_FIXED, dwSize);
+			if (pResCopy != 0) {
+				CopyMemory(pResCopy, pRes, dwSize);
+				VerQueryValue(pResCopy, L"\\", (LPVOID*)&lpFfi, &uLen);
```

Comment:
We're being explicit in the rest of northstar, so do FindResourceW and VerQueryValueW.

Reply:
Resolved in 3643deb64efe15ab38af27bd605b03d587cf1d5b